bundles/org.openhab.binding.guntamatic/src/main/resources/OH-INF/thing/thing-types.xml

```diff
@@ -0,0 +1,596 @@
+				<label>Trigger Warm Wather Circle 1</label>
+			</channel>
+			<channel id="controlWwHeat2" typeId="controlWwHeat">
+				<label>Trigger Warm Wather Circle 2</label>
+			</channel>
+			<channel id="controlExtraWwHeat0" typeId="controlExtraWwHeat">
+				<label>Trigger Extra Warm Wather Circle 0</label>
+			</channel>
+			<channel id="controlExtraWwHeat1" typeId="controlExtraWwHeat">
+				<label>Trigger Extra Warm Wather Circle 1</label>
+			</channel>
+			<channel id="controlExtraWwHeat2" typeId="controlExtraWwHeat">
+				<label>Trigger Extra Warm Wather Circle 2</label>
+			</channel>
+		</channels>
+		<properties>
+			<property name="boilerApproval">K0010</property>
+			<property name="program">PR001</property>
+			<property name="heatCircProgram">HKx01</property>
+			<property name="wwHeat">BKx06</property>
+			<property name="extraWwHeat">ZKx06</property>
+		</properties>
+		<config-description>
+			<parameter name="hostname" type="text" required="true">
+				<context>network-address</context>
+				<label>Hostname</label>
+				<description>Hostname or IP address of the Guntamatic Heating System</description>
+			</parameter>
+			<parameter name="key" type="text" required="false">
+				<context>key</context>
+				<label>Key</label>
+				<description>Optional, but required to read protected parameters and to control the Guntamatic Heating System.
+					The
+					key needs to be reqested from Guntamatic support.</description>
+			</parameter>
+			<parameter name="refreshInterval" type="integer" unit="s" min="1" required="false">
+				<label>Refresh Interval</label>
+				<description>Interval the Guntamatic Heating System is polled in seconds. Default: 60s</description>
+			</parameter>
+			<parameter name="encoding" type="text" required="false">
+				<label>Encoding</label>
+				<description>Code page used by the Guntamatic Heating System. Default: 'windows-1252'</description>
+			</parameter>
+		</config-description>
+	</thing-type>
+
+
+	<channel-type id="controlProgram" advanced="false">
+		<item-type>String</item-type>
+		<label>Set Program</label>
+		<description>Set Program of the Guntamatic Heating System (OFF, NORMAL, WARMWATER, MANUAL)</description>
+		<state>
+			<options>
+				<option value="0">OFF</option>
+				<option value="1">NORMAL</option>
+				<option value="2">WARMWATER</option>
+				<option value="8">MANUAL</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="controlBoilerApproval" advanced="true">
+		<item-type>String</item-type>
+		<label>Set Boiler Approval</label>
+		<description>Set Boiler Approval of the Guntamatic Heating System (AUTO, OFF, ON)</description>
+		<state>
+			<options>
+				<option value="0">AUTO</option>
+				<option value="1">OFF</option>
+				<option value="2">ON</option>
+			</options>
+		</state>
+	</channel-type>
+	<channel-type id="controlHeatCircProgram" advanced="true">
+		<item-type>String</item-type>
+		<label>Set Program of Heat Circle 0...8</label>
```

Comment:
Shoirt label like for example "Heat Circle  Program".

Reply:
see comment from above [#discussion_r776379749](https://github.com/openhab/openhab-addons/pull/11598#discussion_r776379749)

Reply:
You can at least use "Set Heat Circle Program".
Keep in mind that this label is used by MainUI to create item when linking your channel.

Reply:
`<label>Set Heat Circle Program</label>`